docs/welcome/integration.md

```diff
@@ -3,6 +3,7 @@
 
 To start trying out Cpp2 syntax in any existing C++ project, just add a build step to translate the Cpp2 to Cpp1 syntax:
 
+- Copy the `.cpp` file to the same name with a `.cpp2` extension.
```

Comment:
This assumes you're converting an existing file.  If it's a new file, you'd need to add both to the project.